packages/edit-site/src/components/canvas-spinner/index.js

```diff
@@ -3,10 +3,10 @@
  */
 import { Spinner } from '@wordpress/components';
 
-export default function CanvasSpinner() {
+export default function CanvasSpinner( { spinnerStyle } ) {
 	return (
 		<div className="edit-site-canvas-spinner">
```

Comment:
Since we're applying the text color here, what about applying the background color here too? Also, can we call the hooks to get the values here instead?

Reply:
I wanted to keep it generic, but I'm happy to do it here as you suggested.

Reply:
I think we can probably see this one as "the site editor loader" so I think it's fine to have logic to retrieve global styles in it. We do have a more generic component in the components package.

Reply:
> Since we're applying the text color here, what about applying the background color here too?

I think this might require changes to the spinner component itself, so I'll consider working on this in a separate PR if you don't mind.

> Also, can we call the hooks to get the values here instead?

Done in https://github.com/WordPress/gutenberg/pull/51709/commits/929f99db3a08b111a4d7df920df9a22fa66ee718